proto/dbval/dbval.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package ethereum.eth.dbval;
+
+option go_package = "github.com/prysmaticlabs/prysm/v4/proto/dbval;dbval";
+
+message BackfillStatus {
```

Comment:
Can you add comments explaining the meaning of these fields? I would expect both `origin` and `low` to hold  low-end values, but this can't be true.